### ISSUE TITLE
Backspaces in man completion

### ISSUE THREAD

**Issue by xFA25E** [JonWaltman/pcmpl-args.el#6]
Hi, I'm having backspace escape sequences in man completions.

![example](https://raw.githubusercontent.com/xFA25E/images/main/backspaces_.jpg)

It seems to me, that it can be fixed just by piping man output to `col -b`, which is how I'm currently fixing it in my config

```
(setq pcmpl-args-man-function
      (lambda (name)
        (pcmpl-args-default-man-function name)
        (call-process-region (point-min) (point-max) "col" t t nil "-b")))
```

Should I submit a PR for this?

**Comment by CeleritasCelery** [comment]
Thanks for looking into this! Does #4 fix this issue for you?

**Comment by xFA25E** [comment]
Hi, sure, it will fix the issue. But, I think that it would be better if `name` was processed with `shell-quote-argument` first.

**Comment by CeleritasCelery** [comment]
Closed with #4 

Thanks for your help!
